### PR TITLE
Improve documentation and examples for some functions.

### PR DESCRIPTION
**Pull request by Firstyear**
This change improves the examples for setting mailbox capacity,
it improves the documentation of stop and terminate, and adds
some extra comments for clarity in do_send.

Author: William Brown <william@blackhats.net.au>

### CODE REVIEW

File: src/actor.rs [hunk 1]
```diff
@@ -230,13 +230,16 @@ impl ActorState {
 /// actor's communication channels (message handling).
 pub trait ActorContext: Sized {
     /// Immediately stop processing incoming messages and switch to a
-    /// `stopping` state
+    /// `stopping` state. This only affectrs actors that are currently
```
Comment (DoumanAsh) [inline src/actor.rs]: `affectrs` -> `affects`

File: src/actor.rs [hunk 2]
```diff
@@ -230,13 +230,16 @@ impl ActorState {
...
     fn stop(&mut self);
 
-    /// Terminate actor execution
+    /// Terminate actor execution unconditionally. This sets the actor
+    /// into the `stopped` state. This causes future attempts to queue
+    /// messages to fail.
     fn terminate(&mut self);
 
-    /// Actor execution state
+    /// View the current Actor execution state.
```
Comment (DoumanAsh) [inline src/actor.rs]: `View` -> `Get` or `Retrieve`

File: src/address/mod.rs [hunk 3]
```diff
@@ -82,10 +82,10 @@ impl<A: Actor> Addr<A> {
     }
 
     #[inline]
-    /// Sends a message unconditionally.
+    /// Sends a message unconditionally, ignoring any faults that occur.
```
Comment (DoumanAsh) [inline src/address/mod.rs]: `faults that occur` -> `potential errors`

File: src/actor.rs [hunk 4]
```diff
@@ -230,13 +230,16 @@ impl ActorState {
...
     fn stop(&mut self);
 
-    /// Terminate actor execution
+    /// Terminate actor execution unconditionally. This sets the actor
+    /// into the `stopped` state. This causes future attempts to queue
+    /// messages to fail.
     fn terminate(&mut self);
 
-    /// Actor execution state
+    /// Get or retrieve the current Actor execution state.
```
Comment (DoumanAsh) [inline src/actor.rs]: Sorry for causing misudnerstanding, I meant to use either `Get` or `Retrieve` as verb here instead. Not both of them 😄 
Choose what yo like more

Comment (Firstyear) [reply src/actor.rs]: Haha the comment did confuse me, but I decided "nah, I'll let it pass, @DoumanAsh must have something in mind". :)

**Comment by DoumanAsh** [comment]
Thanks

**Pull request merged**